dev-lang/rakudo/rakudo-2022.12.ebuild

```diff
@@ -0,0 +1,85 @@
+# Copyright 1999-2022 Gentoo Authors
```

Comment:
1. You should use `pkgdev commit` - the copyright year is wrong here
2. The commit message format is also wrong (it should be `cat/pkg: add ...`)

Reply:
Is it good now?

Reply:
Resolving it for now.